annotations/annotations_rw.go

```diff
@@ -128,19 +135,24 @@ func (rw *annotationsRW) GTG() error {
 	req, err := http.NewRequest("GET", rw.endpoint+"/__gtg", nil)
 	if err != nil {
 		log.WithError(err).Error("Error in creating the HTTP request to annotations RW GTG")
-		return fmt.Errorf("gtg HTTP request error: %v", err)
+		return fmt.Errorf("%w %v", ErrGTGRequest, err)
```

Comment:
It's standard in golang when wrapping an error to be at the end of the string and be separate by `:` from the rest of the error message. So `"%v : %w"`.